openslides/core/migrations/0003_auto_20160803_1058.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.7 on 2016-08-03 10:58
+from __future__ import unicode_literals
+
+import uuid
+
+from django.db import migrations
+from openslides.core.models import Projector
+
+
+def add_speaker_overlay(apps, schema_editor):
+    """
+    Adds a list of speaker overlay to the existing projector id=1
+    """
+    projector = Projector.objects.get(id=1)
+    soverlay = {}
+    soverlay_uuid = uuid.uuid4().hex
+    soverlay[soverlay_uuid] = {
+        'name': 'core/speakeroverlay',
+        'stable' : True,
+        'uuid' : soverlay_uuid}
+    projector.elements.update(soverlay)
```

Comment:
Just do describe the main points:
- You can not save something to projector.elements but to projector.config. See core/models.py.
- You should not import Projector in global scope but use the model from the versioned app registry.
- You have to disable autoupdate otherwise you would get an error. Here we can think about a better solution: Disable autoupdate globally during migrations, e. g. in a customized migrate command.